Formula/hueadm.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class Hueadm < Formula
+  desc "Command-line management interface to phillips hue"
+  homepage "https://github.com/bahamas10/hueadm"
+  url "https://registry.npmjs.org/hueadm/-/hueadm-1.2.0.tgz"
+  sha256 "8b5936a84f112ce1d1ca9a2a917223755413e3dfa3854088fb4f2c01e2470e72"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_match "1.2.0", shell_output("#{bin}/hueadm -v")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula